en/arena-tag.json

```
@@ -75,8 +75,10 @@
   "TypeImmuneDamageOverTimeOnAddEnemyFIRE": "The opposing Pokemon got surrounded by fire!",
   "TypeImmuneDamageOverTimeOnAddEnemyWATER": "The opposing Pokemon got caught in the vortex of water!",
   "TypeImmuneDamageOverTimeOnAddEnemyROCK": "The opposing Pokemon got surrounded by rocks!",
-  "TypeImmuneDamageOverTimeLapseGRASS": "{{pokemonNameWithAffix}} is hurt by G-Max Vine Lash’s ferocious beating!",
-  "TypeImmuneDamageOverTimeLapseFIRE": "{{pokemonNameWithAffix}} is burning up within G-Max Wildfire’s flames!",
-  "TypeImmuneDamageOverTimeLapseWATER": "{{pokemonNameWithAffix}} is hurt by G-Max Cannonade’s vortex!",
-  "TypeImmuneDamageOverTimeLapseROCK": "{{pokemonNameWithAffix}} is hurt by the rocks thrown out by G-Max Volcalith!"
+  "TypeImmuneDamageOverTimeLapseGRASS": "{{pokemonNameWithAffix}} is hurt by G-Max Vine Lash's ferocious beating!",
+  "TypeImmuneDamageOverTimeLapseFIRE": "{{pokemonNameWithAffix}} is burning up within G-Max Wildfire's flames!",
+  "TypeImmuneDamageOverTimeLapseWATER": "{{pokemonNameWithAffix}} is hurt by G-Max Cannonade's vortex!",
+  "TypeImmuneDamageOverTimeLapseROCK": "{{pokemonNameWithAffix}} is hurt by the rocks thrown out by G-Max Volcalith!",
```

```suggestion
  "TypeImmuneDamageOverTimeLapseGRASS": "{{pokemonNameWithAffix}} is hurt by G-Max Vine Lash’s ferocious beating!",
  "TypeImmuneDamageOverTimeLapseFIRE": "{{pokemonNameWithAffix}} is burning up within G-Max Wildfire’s flames!",
  "TypeImmuneDamageOverTimeLapseWATER": "{{pokemonNameWithAffix}} is hurt by G-Max Cannonade’s vortex!",
  "TypeImmuneDamageOverTimeLapseROCK": "{{pokemonNameWithAffix}} is hurt by the rocks thrown out by G-Max Volcalith!",
```
Give the fancy ’ back! :D